### ISSUE TITLE
Option needed to 'Save' as UFO 

### ISSUE THREAD

**Issue by vernnobile** [fontforge/fontforge#669]
At moment the only file format FontForge can 'Save' as is .sfd, i would deeply love to have the option to save as a .ufo in exactly the same way that i can now save as a .sfd. This would allow fontforge to dovetail seamlessly and conveniently into a workflow with all the other UFO based font creation apps, e.g. robofont, glyphs, metrics machine, superpolator, prepolator, roundingUFO, UFOstretch, metapolator, etc etc

This could either be sone via a keystroke, or by a setting in fontforge preferences, to toggle the default file type between ufo and sfd

:-)

**Comment by marnen** [comment]
@ultrasquid Agreed. If anyone reads this, please reopen. The fundamental issue is still unresolved, right?

**Comment by davelab6** [comment]
@marnen fontforge is basically unmaintained today. Please consider using @trufont instead :)

**Comment by marnen** [comment]
@davelab6 I [did consider it](https://github.com/marnen/borg-sans-mono/issues/10). Unfortunately, TruFont doesn't work very well, at least on Mac OS. FontForge is stil more stable and capable.

**Comment by davelab6** [comment]
TruFont will work better sooner with more contributors :)

**Comment by marnen** [comment]
@davelab6 I'd contribute, but I know very little about developing native desktop applications (I'm a Web developer), and not that much about digital typography either. I'll see what I can do, though...I'd really like it to work.

**Comment by davelab6** [comment]
You might also be interested in @metapolator then :)

**Comment by marnen** [comment]
@davelab6 I might indeed. :)
